CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # Changelog
 
+## 1.0.7 (2020/06/07)
+* Added support for interfaces
+* Removed return value of constructors
```

Comment:
It can be configured: ""phpdoc-generator.constructorWithType": false/"void"/"returnDefinedTypeOrUndefinedType". Default value: false.